sql/ddl_api.sql

```diff
@@ -194,6 +194,12 @@ CREATE OR REPLACE PROCEDURE distributed_exec(
        transactional BOOLEAN = TRUE)
 AS '@MODULE_PATHNAME@', 'ts_distributed_exec' LANGUAGE C;
 
+-- Execute pg_create_restore_point() on each data node
+CREATE OR REPLACE FUNCTION create_distributed_restore_point(
+    name                   TEXT
+) RETURNS TABLE(node_name NAME, restore_point pg_lsn)
```

Comment:
Currently, in the output `node_name IS NULL` implies that it is the access node.
What if we'd be more explicit about this and return an additional column 

`(node_name NAME, node_role TEXT NOT NULL, restore_point pg_lsn)`

OTOH, this is exactly how some pg catalogs also work (`pg_db_role_setting` which if `setrole IS NULL` means it is set for the whole database), so there's some precedent here.

Reply:
I think this is fine, I'll bring this point to discussion with everyone